src/plugins/data_source_management/public/components/data_source_selector/create_data_source_selector.tsx

```diff
@@ -5,10 +5,15 @@
 
 import React from 'react';
 import { IUiSettingsClient } from 'src/core/public';
+import { DataSourcePluginSetup } from 'src/plugins/data_source/public';
 import { DataSourceSelector, DataSourceSelectorProps } from './data_source_selector';
 
-export function createDataSourceSelector(uiSettings: IUiSettingsClient) {
+export function createDataSourceSelector(
+  uiSettings: IUiSettingsClient,
+  dataSource: DataSourcePluginSetup
```

Comment:
```suggestion
  dataSourcePluginSetup: DataSourcePluginSetup
```

Reply:
@xinruiba Fixed the naming, could you take another look?